examples/brainmapper.py

```diff
@@ -1,9 +1,13 @@
 """
-Visualise the output from brainmapper. Cells transformed to atlas space can
-be found at brainmapper_output/points/points.npy.
+Visualise the output from brainmapper.
+Cells transformed to atlas space can be found at
+brainmapper_output/points/points.npy or exported by the brainmapper
+napari
+widget
```

Comment:
```suggestion
napari widget
```

Is the extra new line necessary?

Reply:
Not at all!